base/linalg/ldlt.jl

```diff
@@ -25,6 +25,12 @@ function ldltfact!{T<:Real}(S::SymTridiagonal{T})
     end
     return LDLt{T,SymTridiagonal{T}}(S)
 end
+
+doc"""
+    ldltfact(A) -> F
+
+Compute an `LDLt` factorization `F` of a `SymTridiagonal` matrix `A` such that `A=L*Diagonal(d)*L'` where `L` is a unit lower triangular matrix and `d` is a vector. The main use of an `LDLt` factorization `F=ldltfact(A)` is to solving the linear system of equations `Ax=b` with `F\b`.
```

Comment:
Would be better to be consistent on whether `LDLt` is quoted (see the other doc string in `helpdb.jl`)